### ISSUE TITLE
Enable quarto documents

### ISSUE THREAD

**Issue by AaronGullickson** [hackmdio/hackmd-io-issues#344]
**Is your feature request related to a problem? Please describe.**

I have been using HackMD with great success as a collaboration tool in academic research when use R Markdown documents to write papers. HackMD seems to recognize the .Rmd extension on R Markdown documents and imports them from GitHub without problem. However, Posit (the company that created R Markdown) recently created a second generation R Markdown called Quarto. Like R Markdown, it is a markdown format document with executable code chunks. The file suffix is .qmd which unfortunately does not seem to be recognized by HackMD when trying to import a file from GitHub. 

**Describe the solution you'd like**

Please add the .qmd extension to the list of available markdown documents that one can import from GitHub. 

**Describe alternatives you've considered**

There is not alternative solution that I am aware of. 

**Additional context**

I teach workshops to academics on using R Markdown/Quarto and GitHub and I regularly promote HackMD as a good option for author collaboration when using R Markdown. I would like to be able to do the same for Quarto!

**Comment by tracykteal** [comment]
I agree! It would be very helpful to enable [Quarto](https://quarto.org/) files.

**Comment by AaronGullickson** [comment]
I just want to check in on this. I have many students in my workshops asking me for a collaborative environment for writing quarto documents, and I would like to be able to direct them to HackMD. It seems like it would be a pretty quick fix to add this to the list of editable suffixes, just like .Rmd files currently are.

**Comment by frodew** [comment]
Would love that feature too! This would make HackMD even more attractive for research teams.
